packages/launchpad/src/setup-actions.ts

```diff
@@ -10,6 +10,12 @@ export const setUpActionsForTasks = ( {
 	const { recordTracksEvent, checklistSlug, tasklistCompleted, launchpadContext } = tracksData;
 	const { setShareSiteModalIsOpen } = extraActions || {};
 
+	function mapTask( task: Task, index: number ) {
+		task.order = index;
+	}
+
+	tasks.map( mapTask );
```

Comment:
We're sorting the tasks below on line 34, so this "order" is only internal and doesn't reflect the actual order the user sees. Should we do this in the final return of the function? I am thinking something like the following:
```ts
	// Add actions to sorted tasks.
	return sortedTasks.map( ( task: Task, sortedIndex: number ) => {
		...
		return { ...task, actionDispatch, order: sortedIndex };
	} );